src/lib/mcuboot/include/mcuboot_config/mcuboot_config.h

```diff
@@ -95,7 +95,7 @@
 
 /* Default maximum number of flash sectors per image slot; change
  * as desirable. */
-#define MCUBOOT_MAX_IMG_SECTORS 256
+#define MCUBOOT_MAX_IMG_SECTORS 121
```

Comment:
This is the number of sectors, which should be 968kB(app_size)/8kB(flash page size)